themes/cardinal_service/lib/components/opportunity-row/opportunity_row.html.twig

```diff
@@ -35,6 +42,13 @@
       <h3>
         {{ title }}
       </h3>
+      {% if time_commitment|render|striptags|trim is not empty || service_theme|render|striptags|trim is not empty%}
```

Comment:
I think the tests fail because `||` should be `or`. Can you verify the `/patterns` page doesn't throw an error.